package.json

```diff
@@ -27,19 +27,19 @@
   "author": "",
   "license": "ISC",
   "lint-staged": {
-    "./src/**/*.js": [
+    "'./src/**/*.js'": [
       "prettier --write",
       "eslint --fix"
     ],
-    "./src/**/*.ts": [
+    "'./src/**/*.ts'": [
       "prettier --write",
       "eslint --fix"
     ],
-    "./client/**/*.js": [
+    "'./client/**/*.js'": [
       "cd client && prettier --write",
       "cd client && eslint --fix"
     ],
-    "./client/**/*.ts": [
+    "'./client/**/*.ts'": [
```

Comment:
could you elaborate on these changes?

Reply:
This prettier command doesn't work on Ubuntu without doing this.